src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -535,13 +535,13 @@ public virtual bool CanDeserialize(XmlReader xmlReader)
         }
 
         [RequiresUnreferencedCode(TrimSerializationWarning)]
-        public static XmlSerializer[] FromMappings(XmlMapping[]? mappings)
+        public static XmlSerializer?[] FromMappings(XmlMapping[]? mappings)
```

Comment:
You'll need to update https://github.com/dotnet/runtime/blob/main/src/libraries/System.Xml.XmlSerializer/ref/System.Xml.XmlSerializer.cs as well.